openmc/mgxs_library.py

```diff
@@ -1102,8 +1102,9 @@ def set_nu_fission_mgxs(self, nu_fission, temperature=294.,
 
         """
 
-        check_type('nu_fission', nu_fission, (openmc.mgxs.NuFissionXS,
+        check_type('nu_fission', nu_fission, (openmc.mgxs.FissionXS,
                                               openmc.mgxs.NuFissionMatrixXS))
+        check_value('nu', nu_fission.nu, [True])
```

Comment:
`check_value` requires a `collections.Container` object for the `accepted_values` parameter.